velox/type/tests/DecimalTest.cpp

```diff
@@ -176,5 +176,19 @@ TEST(DecimalTest, toByteArray) {
   testToByteArray(DecimalUtil::kLongDecimalMax, expected8, 16);
 }
 
+TEST(DecimalTest, valueInRange) {
+  ASSERT_TRUE(DecimalUtil::valueInRange<int64_t>(12, 3));
```

Comment:
Ca you add a negative test for precision 3?

1000 and 1234.

Maybe also add a positive test for 999 for completeness.